src/components/sbb-expansion-panel-header/sbb-expansion-panel-header.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Component,
+  ComponentInterface,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  JSX,
+  Prop,
+  State,
+} from '@stencil/core';
+import {
+  actionElementHandlerAspect,
+  createNamedSlotState,
+  HandlerRepository,
+  namedSlotChangeHandlerAspect,
+} from '../../global/eventing';
+import { ButtonProperties, resolveButtonRenderVariables } from '../../global/interfaces';
+
+/**
+ * @slot icon - Slot used to render the panel header icon.
+ * @slot unnamed - Slot used to render the panel header text.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-expansion-panel-header.scss',
+  tag: 'sbb-expansion-panel-header',
+})
+export class SbbExpansionPanelHeader implements ButtonProperties, ComponentInterface {
+  /**
+   * The icon name we want to use, choose from the small icon variants
+   * from the ui-icons category from here
+   * https://icons.app.sbb.ch.
+   */
+  @Prop() public iconName?: string;
+
+  /** Whether the button is disabled . */
+  @Prop({ reflect: true }) public disabled: boolean;
+
+  @Element() private _element!: HTMLElement;
+
+  /** State of listed named slots, by indicating whether any element for a named slot is defined. */
+  @State() private _namedSlots = createNamedSlotState('icon');
+
+  @Event({
+    bubbles: true,
+    eventName: 'toggle-expanded',
+  })
+  public toggleExpanded: EventEmitter;
+
+  private _handlerRepository = new HandlerRepository(
+    this._element,
+    actionElementHandlerAspect,
+    namedSlotChangeHandlerAspect((m) => (this._namedSlots = m(this._namedSlots))),
+  );
+
+  public connectedCallback(): void {
+    this._handlerRepository.connect();
+  }
+
+  public disconnectedCallback(): void {
+    this._handlerRepository.disconnect();
+  }
+
+  private _emitExpandedEvent(): void {
+    if (!this.disabled) {
+      this.toggleExpanded.emit();
+    }
+  }
+
+  public render(): JSX.Element {
+    const { hostAttributes } = resolveButtonRenderVariables(this);
+    hostAttributes['aria-disabled'] = this.disabled?.toString();
```

Comment:
This should already be defined by `resolveButtonRenderVariables`. Why is this necessary?

Reply:
It's not, thanks for pointing out. Removed in 85ed0b4dab8130e456bbd30781133064d83d0056